docs/common-reference-docs/3-configuring-rules.md

```diff
@@ -60,20 +60,64 @@ Collab.Land supports many fungible and non-fungible tokens on many chains. Pleas
 
 ### Token Details Explained:
 
-To find the token details, please refer to this article: [Find Token Details](./4-finding-token-details)
+To find the token details, please refer to this article: [Find Token Details](./finding-token-details)
 
 **chainId***: Enter the chain ID that your token is on.The chainID is a unique ID assigned to each chain(L1 and L2) - For example 1 is the the chainID for Ethereum Mainnet 137 is the chainID of Polygon Mainnet. You can find the chainIds here: https://chainlist.org/
 
-**Type***: Select the token type of your token from the dropdown list.
+**type***: Select the token type of your token. For example, for EVM based chains, it can ERC-20, ERC-721, ERC-1155, etc.
```

Comment:
Some of the properties have an extra (*) that show up in the final content
<img width="159" alt="image" src="https://user-images.githubusercontent.com/20874031/210786376-eb2fcd56-3076-48fb-a556-da614741b242.png">